README.MD

```diff
@@ -234,5 +234,28 @@ The event payload has following structure:
 Kafka partition key for all the events is instance id (for items it is retrieved from
 associated holding record).
 
-*Note for items:* the `new` and `old` records also includes `instanceId` property,
-on the same level with other item properties, which defined in the schema.
+## Domain events for items
+
+The `new` and `old` records also includes `instanceId` property,
+on the same level with other item properties, which defined in the schema:
+```javascript
+{
+  "instanceId": "<the instance id>",
+  // all other properties that defined in the schema
+}
+```
+
+## Domain events for delete all APIs
+
+There are delete all APIs for items instances and holding records. For such
+APIs we're issuing a special domain event:
+* Partition key: `00000000-0000-0000-0000-000000000000`
```

Comment:
What does this partition key do? 

I imagine Kafka still allocates it to a partition and only a single consumer in the group will process it. Is that a sensible understanding?

Reply:
> What does this partition key do?
This is a nil UUID, just to prevent NPEs for consumers. 

This event should be generated only once per tenant life, so it is fine to process it by a single consumer.

Reply:
> This event should be generated only once per tenant life

How do we know this is likely to be the case?

Reply:
Thanks. I think that means that my understanding that this message will be partitioned like other messages and processed by a single consumer is sensible.